README.rst

```diff
@@ -23,20 +23,31 @@ Building is as simple as
 ::
 
         $ git clone https://github.com/FreeRCT/FreeRCT.git
-        $ cd FreeRCT # Go into the downloaded source directory.
-        $ cmake .    # Checks libraries are where they're supposed to be and replaces some strings
-        $ make       # Let make do the heavy work.
+        $ cd FreeRCT    # Go into the downloaded source directory.
+        $ cmake .       # Checks libraries are where they're supposed to be and replaces some strings.
```

Comment:
While it's great this works imho, officially cmake only does out-of-source builds.

It is generally expected that something like
``` bash
$ mkdir build
$ cd build
$ cmake ../FreeRCT
$ make
$ make install
```
works.

If it doesn't, then either make it work or warn against it in the documentation and make it fail if tried anyway.

Perhaps advertise this as an extra nice feature :)

Reply:
Tested it and it works, so I changed the example accordingly.